Tests/BenchmarkTests/BenchmarkCommandTests.swift

```diff
@@ -75,13 +77,68 @@ final class BenchmarkCommandTests: XCTestCase {
         }
     }
 
+    func testParseZeroWarmupIterationsError() throws {
+        do {
+            _ = try BenchmarkCommand.parse(["--warmup-iterations", "0", "--allow-debug-build"])
+            XCTFail("Options successfully parsed when they should not have.")
+        } catch {
+            let message = BenchmarkCommand.message(for: error)
+            XCTAssert(
+                message.starts(
+                    with:
+                        "Value provided via --warmup-iterations must be a positive integer number.")
+            )
+        }
```

Comment:
We're having this pattern more and more... might be good to pull out into a helper function?

Reply:
Refactored in 794916f